File: unittest/scripts/auto/js_devapi/scripts/mysqlx_tableinsert_help.js
//@ Initialization
shell.connect(__uripwd + '/mysql');
var table = db.user;
var crud = table.insert();
session.close();

//@ TableInsert help
crud.help();

//@ TableInsert help, \? [USE:TableInsert help]
\? TableInsert

//@ Help on execute
crud.help('execute');

//@ Help on execute, \? [USE:Help on execute]
\? TableInsert.execute

//@ Help on help
crud.help('help');

//@ Help on help, \? [USE:Help on help]
\? TableInsert.help

//@ Help on insert
crud.help('insert');

//@ Help on insert, \? [USE:Help on insert]
\? TableInsert.insert

//@ Help on values
crud.help('values');

//@ Help on values, \? [USE:Help on values]
\? TableInsert.values

File: unittest/scripts/auto/py_devapi/scripts/mysqlx_tableinsert_help.py
shell.connect(__uripwd + '/mysql')
tableinsert = db.user.insert()
session.close()

#@ tableinsert
tableinsert.help()

#@ global ? for TableInsert[USE:tableinsert]
\? TableInsert

#@ global help for TableInsert[USE:tableinsert]
\help TableInsert

#@ tableinsert.help
tableinsert.help('help')

#@ global ? for help[USE:tableinsert.help]
\? TableInsert.help

#@ global help for help[USE:tableinsert.help]
\help TableInsert.help

#@ tableinsert.insert
tableinsert.help('insert')

#@ global ? for insert[USE:tableinsert.insert]
\? TableInsert.insert

#@ global help for insert[USE:tableinsert.insert]
\help TableInsert.insert

#@ tableinsert.values
tableinsert.help('values')

#@ global ? for values[USE:tableinsert.values]
\? TableInsert.values

#@ global help for values[USE:tableinsert.values]
\help TableInsert.values
